src/backend/wine/manager/downloader/utilities.ts

```diff
@@ -52,6 +52,21 @@ async function fetchReleases({
 
           releases.push(release_data)
         }
+
+        // add latest to list
+        releases.unshift({
+          version: releases[0].version.replace(
+            /[0-9]+[-|.]+[0-9]+/m,
+            '-latest'
```

Comment:
I wonder if writing `${type}-latest` (so for example "Wine-GE-latest") would be easier to read here

Reply:
Depends if we are fine that some information is missing:
- With regex:
![image](https://user-images.githubusercontent.com/61798668/224503176-00b03538-0d5e-4093-9515-3867137eca45.png)

- Your suggestion:
![image](https://user-images.githubusercontent.com/61798668/224503163-7bd7a0fb-55dd-4a1f-a0d8-a9f5c58f5060.png)

I don't mind which to take.

Reply:
Well, the fact that Wine-GE is based on Proton's Wine source is not something the general person has to know, and everyone who does likely already knows this
I just see the potential confusion with GE-Proton there